Samples/ResourceManagement/cs-winforms-unpackaged/winforms_unpackaged_app.csproj

```diff
@@ -5,13 +5,14 @@
     <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <UseWindowsForms>true</UseWindowsForms>
+    <WindowsPackageType>None</WindowsPackageType>
```

Comment:
><WindowsPackageType>None</WindowsPackageType> [](http://example.com/codeflow?start=4&length=45)

curious about this. is this necessary now?

Reply:
Yeah :) As of preview3, this is the new way to load the dependency we needed. As I understand, having this property makes nuget do the rest for you by using module initializers behind the scenes. If the initializer is simple, you can just do it with the tag, or you can keep the method and customize it (which we don't need). In fact, we don't even need the MddBootstrap.cs file anymore, which I forgot to remove. I'll do that as well :)